soccer/gameplay/plays/tuning/test_jittery_motion.py

```diff
@@ -47,7 +47,7 @@ def execute_running(self):
         self.robot.set_world_vel(vel)
 
 
-class TestJitteryMotion(play.Play):
+class TestJitteryMotion(standard_play.StandardPlay):
     def __init__(self):
```

Comment:
These tuning plays shouldn't have defense added to them (I'm guessing either continue to extend from Play, or add an option somehow to turn off defense in defense).